onnxscript/type_annotation.py

```diff
@@ -143,6 +166,85 @@ def get_return_types(typeinfo: type | typing.Sequence[type]) -> typing.Sequence[
     """
     if isinstance(typeinfo, typing.Sequence):
         return typeinfo
-    if get_origin(typeinfo) is tuple:
-        return get_args(typeinfo)
+    if typing.get_origin(typeinfo) is tuple:
+        return typing.get_args(typeinfo)
     return (typeinfo,)
+
+
+def pytype_to_input_strings(pytype: TypeAnnotationValue) -> list[str]:
+    """Returns a list of all supported input types in string representation for a given type annotation.
```

Comment:
Nit: maybe `Returns a list of type-strings corresponding to a given type annotation`? It could represent even an output-type, not just input-type.

Reply:
Done